package.json

```diff
@@ -46,6 +46,7 @@
   ],
   "devDependencies": {
     "@babel/core": "^7.4.3",
+    "@babel/plugin-proposal-optional-chaining": "^7.2.0",
```

Comment:
It doesn't look like this plugin is actually being used (other than set up in the babelrc above). Can it be removed?

Reply:
Will do!